source/uwp/winui3/AdaptiveCardsObjectModel/idl/AdaptiveCards.ObjectModel.Winui3.idl

```diff
@@ -0,0 +1,864 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace AdaptiveCards.ObjectModel.Winui3
+{
+    runtimeclass AdaptiveCardParseResult;
+
+    enum TextSize
+    {
+        Small = 0,
+        Default,
+        Medium,
+        Large,
+        ExtraLarge
+    };
+
+    enum TextWeight
+    {
+        Lighter = 0,
+        Default,
+        Bolder,
+    };
```

Comment:
There are inconsistencies between enums finishing with a comma and others don't have it, can you add a comma to all of them?

Reply:
Done.